src/idiom284.fuzion

```diff
@@ -0,0 +1 @@
+a := array n i->0
```

Comment:
The standard list type in Fuzion is `Sequence`. `array` inherits from `Sequence`, so this is ok, but very inefficient since it creates a (possibly large) array. 
There are several lazy alternatives, that work well for large `n`, e.g., 
```
  a := [0]*n
```
```
  a := (0 : nil)*n
```
```
  a := (0 :: x->x).take n
```
```
  zeros := 0 : zeros
  a := zeros.take n
```

Reply:
The first two examples are not working for me.

The first one
```
ex is
  a := [0]*n
  say a
```
produces this
```
/tmp/zeros.fz:2:11: error 1: Could not find called feature
  a := [0]*n
----------^
Feature not found: 'infix *' (one argument)
Target feature: 'array'
In call: '*'

one error.
```

And for the second one it's the same except for `Target feature: 'list'`.

Reply:
I think this is only defined for String right now.
```
# repeat string given number of times
public infix * (n i32) ref : String
```
but  this should work

```
[0].cycle.take n
or
(0 : nil).cycle.take n
```